demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -25,7 +25,7 @@
    "source": [
```

Comment:
<p>Strangely, this isn't rendering correctly on ReviewNB, but it is on Github and RtD, so I guess it's not a problem. As a double-check, I assume it renders correctly in a local Jupyter instance?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Yeah it's rendering correctly in my jupyter lab
![image](https://user-images.githubusercontent.com/33508488/83225518-5251ab00-a1c3-11ea-9d78-49626df23f0e.png)

but yeah I've also noticed sometimes ReviewNB doesn't render things correctly (I often see random new lines)